src/PrestaShopBundle/DependencyInjection/Compiler/ContainerInjectionPass.php

```diff
@@ -66,7 +67,7 @@ public function process(ContainerBuilder $container)
 
             $isContainerAware = (
                 $reflectedClass->implementsInterface(ContainerAwareInterface::class)
-                || is_subclass_of($class, AbstractController::class)
+                || is_subclass_of($class, FrameworkBundleAdminController::class)
```

Comment:
Since `FrameworkBundleAdminController` implements `ContainerAwareInterface` this second condition can be removed no? Unless we revert the condition and make `FrameworkBundleAdminController` the only condition when we force to inject the global container?